components/go/pkg/chef_licensing.go

```diff
@@ -0,0 +1,7 @@
+package cheflicensing
+
+import keyfetcher "github.com/chef/chef-licensing/components/go/pkg/key_fetcher"
+
+func FetchAndPersist() []string {
```

Comment:
Might require to handle error if GlobalFetchAndPersist() returning error
```
keys, err := keyfetcher.GlobalFetchAndPersist()
	if err != nil {
		log.Printf(err)
		return nil, err
	}

Reply:
The idea is that the GlobalFetchAndPersist will return the license keys only if it can find them. If there are any errors, the application will exit.